doc/modelling/index.rst

```diff
@@ -0,0 +1,10 @@
+.. _modelling-overview:
+
+Modelling
+=========
+
+Arbor is a new project, and our first serious users are getting their hands dirty writing the first non-toy models. With time, best practices, tips and tricks and a collection of howtos for common features should become apparent, and will then be shown here.
```

Comment:
> Arbor is feature complete for real world modeling, and the user community is busy collaborating with us to build complex models. Best practices, tips, tricks and howtos will be documented here as the community gains experience. 

Is better marketing, and closer to the truth!

Reply:
Weeeellllll, feature complete is a very strong claim, but it's better marketing for sure!

Reply:
Well, it is not possible to ever be feature complete, because there are always more features that can be added. However Arbor has sufficient features to be used for modern complex models (if not all of them).